be/src/common/config.h

```diff
@@ -929,6 +929,9 @@ CONF_mBool(allow_invalid_decimalv2_literal, "false");
 // data page size for primary key index.
 CONF_Int32(primary_key_data_page_size, "32768");
 
+// the max package size be thrift server can receive,avoid accepting error or too large package causing OOM,default 20M
+CONF_Int32(be_thrift_max_pkg_size, "20000000");
```

Comment:
rename to be_thrift_max_pkg_bytes   so that the user will know that it is bytes or Mb

Reply:
I have renamed be_thrift_max_pkg_size to be_thrift_max_pkg_bytes